README.md

```diff
@@ -327,62 +334,56 @@ module.exports = {
 
 ```js
 // src/index.js
 import 'whatwg-fetch';
 ```
 
-### Using @babel/polyfill
-
-Babel allows you to automatically inject the polyfills your application needs based on your [`browserslist`](#browserslist). If you want to use this feature you would first need to disable the [`polyfill`](#polyfill) option to prevent double importing of the same polyfill.
-
-To use this feature install [`@babel/polyfill`](https://babeljs.io/docs/en/babel-polyfill):
-
-```shell
-npm install --save @babel/polyfill
-```
+### Using `@babel/polyfill`
 
-And then place `@babel/polyfill` at the **top of your [`jsEntry`](#jsentry) file.**
-
-```js
-// Using @babel/polyfill
-// src/index.js
-import '@babel/polyfill';
-```
+If you don't care about bundle size and want to polyfill all standard JS, you can install [`@babel/polyfill`](https://babeljs.io/docs/en/babel-polyfill) and import it in your [`jsEntry`] file.
 
 **Warning:** [`polyfill`](#polyfill) must be set to `false` to use `@babel/polyfill` and you should only import `@babel/polyfill` once and only once in your application.
 
 ## Deployment environments
 
 ### Using environment variables
 
-Underreact allows you to inject environment variables during the build time. You can set them up by using the [`environmentVariables`](#environmentvariables) option in your configuration.
+Underreact allows you to inject environment variables into your client-side code at build time. You can set them up by using the [`environmentVariables`](#environmentvariables) option in your configuration.
+
+```js
+// underreact.config.js
+module.exports = {
+  environmentVariables: {
+    SERVER_URL: 'https://ketchup.com'
+  }
+};
+```
 
-**Note: `DEPLOY_ENV` & `NODE_ENV` are special environment variables in Underreact and they should never be set in Underreact configuration.**
+**Note: `DEPLOY_ENV` & `NODE_ENV` are special environment variables in Underreact, so cannot be set in Underreact configuration.**
 
-There are certain things to keep in mind:
+### `DEPLOY_ENV` and `NODE_ENV`
 
-- If `DEPLOY_ENV` is not set it will default to `development` in [development mode](#modes) and `production` in [production mode](#modes).
-- It is recommended that you do not change the default value of `DEPLOY_ENV` when running in [development mode](#modes).
-- `DEPLOY_ENV` is generally be set to `staging` or `production` for production mode, but you can set it to any value you wish to better align with your target environments.
-- `DEPLOY_ENV` is not the same as `NODE_ENV`, refer to [`Why not use NODE_ENV`](#why-not-use-node_env).
+- `NODE_ENV` will default to `'development'` in [development mode](#modes) and `'production'` in [production mode](#modes).
+- By default, `DEPLOY_ENV` is set to `staging` in development mode and `production` in production mode, but you can set it to any value you wish to better align with your target environments (e.g. `DEPLOY_ENV=something npx underreact build`).
```

Comment:
Right now the `DEPLOY_ENV` falls back to the `mode`, which in development mode would be `development` and not `staging`. Could you explain why we would want `staging` as the default `DEPLOY_ENV` for development mode.

Reply:
I misunderstood.

Reply:
Added a commit trying to fix this.